docs/3.api/1.composables/use-async-data.md

```diff
@@ -6,7 +6,7 @@ description: useAsyncData provides access to data that resolves asynchronously.
 Within your pages, components, and plugins you can use useAsyncData to get access to data that resolves asynchronously.
 
 ::alert{type=warning}
-[`useAsyncData`](/docs/api/composables/use-async-data) is a composable meant to be called directly in a setup function, plugin, or route middleware. It returns reactive composables and handles adding responses to the Nuxt payload so they can be passed from server to client without re-fetching the data on client side when the page hydrates.
+[`useAsyncData`](/docs/api/composables/use-async-data) is a composable meant to be called directly in a [Nuxt context](/docs/guide/going-further/nuxt-app#the-nuxt-context) scope. It returns reactive composables and handles adding responses to the Nuxt payload so they can be passed from server to client without re-fetching the data on client side when the page hydrates.
```

Comment:
I would suggest `xxx is a composable meant to be called directly within the Nuxt context` eliminating the need for the term "scope" since it's not present in the explanation of `## The Nuxt context`. 
"a Nuxt context" implies the existance of multiple contexts so using "the" instead would make things even more clear I feel like.

Reply:
Hmm, it's possible, the question is won't it be too "restricting" if we change wording to `the` considering there are multiple available scopes (like plug-ins, middleware, setup functions, etc), and won't it be better to add the scope word in the nuxt context section (like `the possible Nuxt context scopes are ...`)?
Would love to hear your opinion about it!

Reply:
I guess this comes down to technicalities.

From my understanding and mental model there is only one nuxt context at a time hence `the`.

Scopes with how the word is used currently to me implies that the nuxt context is scoped to different parts of the app and so differs between them. 
I think it'd make sense to be careful with the wording so that the programmers "scope" and physical location "scope" don't get mixed here and we don't introduce another technical term to the already existing "context" and "instance" that users need to understand

Reply:
I see what you mean. Do you think we could perhaps combine the 2 and say something like that the nuxt context is the same context, but is accessible from multiple parts of the app? We think of a better phrasing for that, because while it is the same context, it's noteworthy that you can reach it from multiple areas.

Reply:
I think the introduction you've added "The nuxt context" covers that pretty well already.
"Currently, the Nuxt context exists only in plugins, Nuxt hooks, Nuxt middleware, and setup functions (in pages and components)."
To align it with your suggestion it could perhaps be 
"Currently, the Nuxt context is only accessible in plugins, Nuxt hooks, Nuxt middleware and setup functions (in pages and components)."

Reply:
Good idea! I'll add that, thanks!

Reply:
I have also rephrases it accordingly so users will know what "scopes" mean in the given context (no pun intended). That was I feel we could make it clear that while it's a shared context, the scopes refer to the areas in your app that can access it (so scopes in physical definition, rather than the functional definition). Let me know what you think of it!

Reply:
Nice 👍
Personally I feel like I'd just avoid the introduction of "nuxt context scope" alltogether. That way there's less things a user needs to read and understand and less confusion about terms. Especially given that vue also uses scopes (explictly) for a similar ish concept. https://vuejs.org/api/reactivity-advanced.html#effectscope. 
I just think that throwing another term into an already good explanation has little upside but introduces more potential for confusion.
But that's super nitpicky and it's only one sentence so both ways work fine of course :)

Reply:
How would it make it clear where it's usable then? Saying it's only usable in the nuxt context?

Reply:
Well currently that's explained already, right?
> These include plugins, Nuxt hooks, Nuxt middleware, and setup functions (in pages and components).

So I feel like

"The Nuxt context is only accessible in certain parts of your application, specifically from plugins, Nuxt hooks, middleware, and setup functions in pages and components."

stresses context is not accessible in all places, lists the areas it is accessible from and doesn't introduce any new words that would require second thought. 
Perhaps the listing of places with context could also link out to the documentation so plugins links to /plugins etc.

Reply:
Done! Added all the links, if there's any more feedback I'd love to hear it!

Reply:
Nice 👍
Only thing I noticed is that useRouter https://github.com/nuxt/nuxt/pull/23546/files#diff-8637761ca22caaa4a8a782eef64f7da236d0b7c412ec64f308f604f6cbed4263R8 still uses the term scope, but LGTM

Reply:
Oh oops!

Reply:
Fixed it! Thanks for catching it!